package.json

```diff
@@ -156,6 +156,7 @@
     "mini-css-extract-plugin": "^0.8.0",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "postcss-aspect-ratio": "^1.0.0",
+    "postcss-assets": "^5.0.0",
```

Comment:
@ryanurban I tried to use `postcss-url` first, but for the life of me could not get the options configured in a way that worked for our purposes. Probably spent ~3-4 hours playing with the configuration unsuccessfully before trying this as an alternative. 

Honestly, I like this plugin a little better because, as the developer, I have to make a conscience choice to use the plugin's `replace()` function instead of the traditional `url()` syntax. This mean no chance of accidental resolution/replacement/rebasing of a path that shouldn't be changed. (Just my 2-cents.)

Reply:
Awesome, makes sense ⚡️